<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveMappingWidget</class>
 <widget class="QWidget" name="InteractiveMappingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interactive Mapping</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabNA">
      <attribute name="title">
       <string>NA Statistics</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabHMF">
      <attribute name="title">
       <string>HMF Info</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabMAP">
      <attribute name="title">
       <string>Mapping Info</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
